### PR TITLE
The HTML 5 doctype is <!DOCTYPE html>, not <!doctype html>. Also, you forgot to do document.getElementById();

### PR DESCRIPTION
**Pull request by AwesomeWebMaster10**
…getElementById();

### CODE REVIEW

File: test.html [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
   <head>
-    <meta charset="utf8">
+    <meta charset="utf-8">
     <title>Processing.js</title>
     <script src="processing.js"></script>
   </head>
   <body>
     <h1>Processing.js</h1>
     <script type="text/processing">
+      document.getElementById("canvas");
```
Comment (kevinbarabash) [inline test.html]: why is this necessary?

File: test.html [hunk 2]
```diff
@@ -1,13 +1,15 @@
-<!doctype html>
+<!DOCTYPE html>
```
Comment (kevinbarabash) [inline test.html]: I think this is okay, see https://www.w3.org/TR/html5/syntax.html#the-doctype.

**Comment by Pomax** [comment]
common misconception, In XHTML parsing, "DOCTYPE" *must* be uppercase, but for HTML parsing, it's case-insensitive (see https://dev.w3.org/html5/html-author/#html-and-xhtml-comparison)

File: test.html (hunk 1 above)
Comment (Pomax) [inline test.html]: I'd like to know, too, this call does nothing - it gets an element by `id`, and then throws it away again.

File: test.html (hunk 2 above)
Comment (Pomax) [inline test.html]: Yeah this is a common misconception. Under XHTML parsing rules the "DOCTYPE" keyword *must* be uppercase, but under HTML (and modern HTML5) parsing rules, it's case-insensitive (see https://dev.w3.org/html5/html-author/#html-and-xhtml-comparison)

File: test.html [hunk 3]
```diff
@@ -32,6 +34,6 @@
         println(mouseX, mouseY);
       }
     </script>
-    <canvas></canvas>
+    <canvas id="canvas"></canvas>
```
Comment (Pomax) [inline test.html]: this is not necessary. Without a target, Processing.js ties code it sees to the closest sibling `<canvas>` element

**Comment by Pomax** [comment]
If you want to change this PR to just a fix for the `utf-8` meta charset, I'll be happy to merge this in, but the other changes are unnecessary.

**Comment by AwesomeWebMaster10** [comment]
OK, sorry, my bad. I'm new to GitHub.

**Comment by Pomax** [comment]
no worries, we've all been new at some point. Let me know if you want to fix up this PR to be just the utf-8 fix, or whether you'd like to close this PR and then many do the `utf8`->`utf-8` fix as a new, separate fix

**Comment by Pomax** [comment]
superceded by https://github.com/processing-js/processing-js/pull/236
